tasks/main.yml

```diff
@@ -21,6 +23,19 @@
   command: "openssl dhparam -out /etc/prosody/certs/dh-{{ prosody_dhparam_length }}.pem {{ prosody_dhparam_length }} creates=/etc/prosody/certs/dh-{{ prosody_dhparam_length }}.pem"
   notify: restart prosody
 
+- name: Get latest external prosody modules
```

Comment:
Please follow the style of the project, i.e. all parameters in one line (I know it's not the best style but I think it's best to be consist).